make/common/NativeCompilation.gmk

```diff
@@ -438,32 +439,35 @@ define SetupNativeCompilation
 
   $1_EXTRA_LDFLAGS += $(SYSROOT_LDFLAGS)
 
+  # Need to make sure TARGET is first on list
   $1 := $$($1_TARGET)
-  ifneq (,$$($1_LIBRARY))
-    # Generating a dynamic library.
-    $1_EXTRA_LDFLAGS+=$$(call SET_SHARED_LIBRARY_NAME,$$($1_BASENAME))
-    ifeq ($(OPENJDK_TARGET_OS), windows)
-      $1_EXTRA_LDFLAGS+="-implib:$$($1_OBJECT_DIR)/$$($1_LIBRARY).lib"
+  ifeq ($$($1_STATIC_LIBRARY),)
+    ifneq ($$($1_DEBUG_SYMBOLS),)
+      ifeq ($(ENABLE_DEBUG_SYMBOLS), true)
+        ifneq ($(OPENJDK_TARGET_OS), macosx) # no MacOS X support yet
+          ifneq ($$($1_OUTPUT_DIR),$$($1_OBJECT_DIR))
+            # The dependency on TARGET is needed on windows for debuginfo files
+            # to be rebuilt properly.
+            $$($1_OUTPUT_DIR)/% : $$($1_OBJECT_DIR)/% $$($1_TARGET)
+		$(CP) $$< $$@
```

Comment:
There are many unnecessary whitespace changes here relative to:
* 8025936: Windows .pdb and .map files does not have proper dependencies setup

Reply:
Fixed.